pages/common/dotnet.md

```diff
@@ -0,0 +1,19 @@
+# dotnet
+
+> Cross Platform .NET Command Line Tools for .NET Core
+
+- Initialize a new .NET project:
+
+`dotnet new project-type`
```

Comment:
shouldn't project-type be the project name ? What is project-type supposed to mean ?

Reply:
I'll change `project-type` to `template-short-name`. Sounds more correct.

Here's the possible values it can take:
<img width="380" alt="dotnet-new" src="https://user-images.githubusercontent.com/10794213/32118762-7b71d76c-bb70-11e7-97d9-78a59b874489.png">

I would have added them to the example command if they were just a few.

Reply:
I meant they need to be wrapped in tokens. I have added them now.